package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "extension-create": "cross-env NODE_ENV=development node cli.js",
     "lint": "eslint --config=eslintrc.json",
-    "preinstall": "sh preinstall.sh",
+    "preinstall": "cd create && yarn install && cd .. && cd develop && yarn install && cd ..",
```

Comment:
That's a smart move! Works great on macOS too!

Could you also remove the file called on [line 32](https://github.com/cezaraugusto/extension-create/pull/14/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L32)? So we don't break new releases.

Reply:
done